ios/PurchasesHybridCommon/PurchasesHybridCommon/RCErrorContainer.m

```diff
@@ -27,10 +27,11 @@ - (instancetype)initWithError:(NSError *)error extraPayload:(NSDictionary *)extr
         dict[@"underlyingErrorMessage"] = @"";
     }
 
-    if (error.userInfo[RCReadableErrorCodeKey]) {
-        NSString *readableErrorCode = error.userInfo[RCReadableErrorCodeKey];
+    // JOSH: does RCReadableErrorCodeKey become RCPurchasesErrorCodeDomain
+    if (error.userInfo[RCPurchasesErrorCodeDomain]) {
+        NSString *readableErrorCode = error.userInfo[RCPurchasesErrorCodeDomain];
         dict[@"readableErrorCode"] = readableErrorCode;
-        dict[RCReadableErrorCodeKey] = readableErrorCode;
+        dict[RCPurchasesErrorCodeDomain] = readableErrorCode;
```

Comment:
This substitution didn't real right. Should `RCReadableErrorCodeKey` ➡️ `RCPurchasesErrorCodeDomain`?

Reply:
No, the domain isn't a user info key. I think this can be removed? The readable error is already added in the line above.